kubectl-cloudflow/cloudflowavro/avrocompatibility.go

```diff
@@ -0,0 +1,55 @@
+package cloudflowavro
+
+import (
+	"github.com/hamba/avro"
+	"github.com/linkedin/goavro/v2"
```

Comment:
Using this library for schema validation as it seems to be widely used and well maintained. I am not sure yet how it compares with `https://github.com/hamba/avro` when it comes to avro complience but one thing that it lacks is [compatibility checks](https://github.com/linkedin/goavro/issues/200). Thus for compatibility checks I used `https://github.com/hamba/avro`.